erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py

```diff
@@ -406,22 +390,6 @@ def get_tax_accounts(item_list, columns, company_currency,
 				})
 
 	tax_columns.sort()
-	for desc in tax_columns:
-		columns.append({
-			'label': _(desc + ' Rate'),
-			'fieldname': frappe.scrub(desc + ' Rate'),
-			'fieldtype': 'Float',
-			'width': 100
-		})
-
-		columns.append({
-			'label': _(desc + ' Amount'),
-			'fieldname': frappe.scrub(desc + ' Amount'),
-			'fieldtype': 'Currency',
-			'options': 'currency',
-			'width': 100
-		})
-
```

Comment:
@harshmule-git why did you remove this chunk of code and not the entire function. because this called function does not have calling function

Reply:
will remove that!